### PR TITLE
Lib comparison update

### PR DESCRIPTION
**Pull request by Lulalaby**
- Added select menu v2 as row
- Checked through all libraries and updated the information about feature, issues and prs

### CODE REVIEW

**Comment by Lulalaby** [comment]
If selectMenuv2 isn't wanted, I can remove it again.
But at least the updated information about libs is useful

**Comment by BobDotCom** [comment]
Instead of adding a new section for selects v2, maybe update the existing select menus section:
Partial = Support for selects v1
Full = Support for selects v1 & v2

**Comment by Lulalaby** [comment]
Good shot, might be better.
Waiting for blobs opinion.

**Comment by advaith1** [comment]
yeah let's just update the existing column, takes up less space

**Comment by Lulalaby** [comment]
will do

**Comment by Lulalaby** [comment]
This might work?

**Comment by Lulalaby** [comment]
@advaith1 how about a merge :p

File: libs.ts [hunk 1]
```diff
@@ -148,15 +151,9 @@ export const libs: Lib[] = [
 		timeouts: 'Yes',
 		modals: 'Yes',
 		permsv2: 'Yes',
-		automod: {
-			text: 'Has a PR',
-			url: 'https://github.com/Remora/Remora.Discord/pull/203'
-		},
+		automod: 'Yes',
```
Comment (AraHaan) [inline libs.ts:154]: I just updated this in my own PR.

File: libs.ts [hunk 2]
```diff
@@ -148,15 +151,9 @@ export const libs: Lib[] = [
...
 		localization: 'Yes',
-		forums: {
-			text: 'Has a PR',
-			url: 'https://github.com/Remora/Remora.Discord/pull/195'
-		}
+		forums: 'Yes'
```
Comment (AraHaan) [inline libs.ts:156]: And this too.

**Review by AraHaan — CHANGES_REQUESTED** [review]

**Comment by Lulalaby** [comment]
@advaith1 can we get this merged?
I saw you merged the other prs recently and it'll probably introduce more merge conflicts if others get merged first

**Comment by advaith1** [comment]
imo it would be fine to just remove the asterisks and the note; what do u think?

**Comment by Lulalaby** [comment]
I honestly think a few metadata infos aren't bad. Your comparison site is pretty known already and I think better than discords. 

It's good to know exact information about all the libs. I would even like to see to go further to have a reliable source for deciding which lib to use, especially for new devs. 

Hope you get what I mean.

**Comment by Lulalaby** [comment]
yeah. I'm gonna figure something out and respond later with my other idea

**Comment by Lulalaby** [comment]
Neato, thx blob!

**Pull request merged**